examples/dynamo/eg6/alg.x90

```diff
@@ -1,3 +1,14 @@
+! Modifications copyright (c) 2017, Science and Technology Facilities Council
+!-------------------------------------------------------------------------------
+! (c) The copyright relating to this work is owned jointly by the Crown, 
+! Met Office and NERC 2014. 
+! However, it has been created with the help of the GungHo Consortium, 
+! whose members are identified at https://puma.nerc.ac.uk/trac/GungHo/wiki
+!-------------------------------------------------------------------------------
```

Comment:
Again, we should take this opportunity to replace this copyright notice with the full BSD license text.

Reply:
Replaced old copyright notice with the full BSD license.

Reply:
Corrected BSD license as agreed.